src/universal/components/ListFilterSet/ListFilterSet.js

```diff
@@ -3,6 +3,8 @@ import cssModules from 'react-css-modules';
 import styles from './ListFilterSet.scss';
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
My thoughts exactly! 👍 

The disables are just a bandage until we abandon `react-css-modules`.